fbpic/particles/particles.py

```diff
@@ -434,6 +432,10 @@ def push_p( self ) :
         half-timestep *behind* the positions (x, y, z), and it brings
         them one half-timestep *ahead* of the positions.
         """
```

Comment:
The changes below allow to have photon macroparticles in the code (q=0): these macroparticles should not deposit charge, nor should they have their momentum modified or gather the fields.